clang/lib/Basic/Version.cpp

```diff
@@ -123,4 +123,13 @@ std::string getClangFullCPPVersion() {
   return buf;
 }
 
+llvm::SmallVector<std::pair<llvm::StringRef, llvm::StringRef>, 2>
+getSYCLVersionMacros(const LangOptions &LangOpts) {
+  if (LangOpts.getSYCLVersion() == LangOptions::SYCL_2017)
+    return {{"CL_SYCL_LANGUAGE_VERSION", "121"},
+            {"SYCL_LANGUAGE_VERSION", "201707"}};
+  else if (LangOpts.getSYCLVersion() == LangOptions::SYCL_2020)
```

Comment:
```suggestion
  if (LangOpts.getSYCLVersion() == LangOptions::SYCL_2020)
```
`else` is redundant here.
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Reply:
Thanks! Done!